addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/model/OshiSysteminfo.java

```diff
@@ -393,6 +427,20 @@ public DecimalType getSwapAvailablePercent() {
         }
     }
 
+    @Override
+    public DecimalType getSwapUsedPercent() {
+        long usedSwap = memory.getSwapUsed();
+        long totalSwap = memory.getSwapTotal();
+        BigDecimal usedPercent;
+        if (totalSwap > 0) {
```

Comment:
I want to prevent Division by zero. Counter-question: Is totalSwap < 0 possible?